api.planx.uk/modules/analytics/metabase/collection/getTeamAndMetabaseId.ts

```diff
@@ -0,0 +1,43 @@
+import { gql } from "graphql-request";
+import { $api } from "../../../../client/index.js";
+
+interface GetMetabaseId {
+  teams: {
+    id: number;
+    name: string;
+    metabaseId: number | null;
+  }[];
+}
+
+export const getTeamAndMetabaseId = async (name: string) => {
```

Comment:
I think this is fine as one gql request, maybe I'd change the function name to be `getTeamIdAndMetabaseId` just to be completely explicit